client/dataManager.js

```diff
@@ -0,0 +1,29 @@
+class dataManager {
+    constructor() {
+        this.data
+        this.dream
+    }
+
+    players() {
```

Comment:
maybe name it `getPlayers` better than just `players` - it's important to name your functions and methods and variables in a way that the reader understands what they do